app/Http/Controllers/Api/V1/UserLmsSettingsController.php

```diff
@@ -27,6 +27,12 @@ public function __construct(UserLmsSettingsRepositoryInterface $repo)
      */
     public function index()
     {
+        if ($this->repo->all() == '[null]') {
+            return response([
+                'errors' => ['LMS settings not found for this user.'],
```

Comment:
Just double checking on this one. Whether it should be 'null' or [null] - @fahad-curriki

Reply:
Waleed confirmed that its [null] string, so acknowledging!